lundgrea/PalettePicker-FE
# Initial config app

#### What's this PR do?
This PR sets up the initial CSS configurations for our app per the wireframes from yesterday (10/12)
#### Where should the reviewer start?
The reviewer should start in the App folder, looking at the js and css, and do the same for the Sidebar, FoldersForm, FoldersContainer, and Header folders.
#### How should this be manually tested?
As there is not test functionality yet, viewing at localhost:3000 would be the place to see the components at work
#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
'Initial DOM Setup'
#### Screenshots (if appropriate):
Initial wireframes (10/11)
![IMG_0034](https://user-images.githubusercontent.com/11339301/66704895-81bbb200-ecdd-11e9-8fbb-2d38e3e6abf7.JPG)
![IMG_0039](https://user-images.githubusercontent.com/11339301/66704897-841e0c00-ecdd-11e9-82d4-3c3cdfa1d702.JPG)
Current DOM Setup:
![Screen Shot 2019-10-12 at 10 42 56 AM](https://user-images.githubusercontent.com/11339301/66704902-939d5500-ecdd-11e9-9309-aef899d62407.png)
![Screen Shot 2019-10-12 at 10 42 32 AM](https://user-images.githubusercontent.com/11339301/66704906-95671880-ecdd-11e9-84eb-335521a468af.png)

#### Questions:
